### DISCUSSION TITLE
import from offline vcenter

### DISCUSSION THREAD

**Discussion by jclbc** [bb-Ricardo/netbox-sync#414] [Ideas]
Hi,
thank you for this great job.
I need to use the script from offline vcenter, dou you think it could be possible to extract a config file and use it as done for redfish?
Regards,

**Comment by bb-Ricardo** [comment]
Hi,

thank you. I don't really understand the use case. Which part is offline? Can you describe it a bit more detailed?

**Reply by jclbc** [reply]
I want to import isolated vmware clusters (managed in local vcenter) thoses servers are physicaly isolated without interconnetion
if I could:
-run a script in the isolated network to collect vcenter info into a file
-fetch the file
-import the file in the netbox server and run the script to import in netbox

I hope I'm clear.
Thanks

**Reply by bb-Ricardo** [reply]
Thank you, now I understand your use case.

The script does not support this use case. However, you could do it the other way around.

use `govc` to export the venter config and spin up `vcsim` loaded with the config zu simulate the vCenter. But `vcsim` does not support that API to query tags.

**Reply by jclbc** [reply]
Sorry for the delay, Many thanks govc/vcsim works perfectly (and your script!!!)!

**Reply by bb-Ricardo** [reply]
Awesome, you are welcome
